src/patterns/feedback/index.njk

```diff
@@ -0,0 +1,52 @@
+---
+title: Feedback
+group: Ask users for...
+---
+{% from "views/partials/example/_macro.njk" import patternlibExample %}
+{% from "components/panel/_macro.njk" import onsPanel %}
+{% from "components/external-link/_macro.njk" import onsExternalLink %}
+
+## When to use this pattern
+Use this pattern whenever you need to prompt your users to give feedback about a live service.
+
+Conduct user research to find out where your users would most likely want to provide feedback. This is typically at the end of a mandatory questionnaire, after they have submitted their answers.
+
+## When not to use this pattern
+Don't ask users for feedback on every page or in the middle of a transaction or mandatory questionnaire they need to complete. User research studies found users prefer to give feedback at the end of a questionnaire, rather than be interupted during it. They don't expect a response so wouldn't give feedback to report technical issues.
```

Comment:
```suggestion
Don't ask users for feedback on every page or in the middle of a transaction or mandatory questionnaire they need to complete. 

User research studies found users prefer to give feedback at the end of a questionnaire, rather than be interupted during it. They don't expect a response, so wouldn't give feedback to report technical issues.
```